pkg/internal/server/grpc/server.go

```diff
@@ -38,7 +38,8 @@ import (
 )
 
 const (
-	HandlerTraceLogLevel = 4
+	HandlerTraceLogLevel         = 4
+	HandlerDetailedTraceLogLevel = 5
```

Comment:
I added a detailed trace level for the contents of the stream response loop.

Reply:
@carlbraganza I think the logs level is not specific to handler methods, right? I mean the level can be set in log in any package. Should we give generic names like - debug, info, verbose, etc?

Reply:
Klog uses a concept of a process wide verbosity level (the `V` function), and each -v flag increments the level used.   There are no clear definitions for info, debug, etc. - these are "severity" values in the log which could be logged at any verbosity level.

I defined these constants for the handler package, with the only constraint being that 
```
1 <= HandlerTraceLogLevel < HandlerDetailedTraceLogLevel
```
We could define common process wide constants to assign values to these constants later if we see the need, but for now I think most of the business logic is concentrated in the `grpc/server` package.